doc/ref_cert/lfn/chapters/chapter01.md

```diff
@@ -17,62 +17,37 @@
-- Empirical Validation with Reference Golden VNFs (Termed Validation by CVC) will ensure the NFVI runs with a set of VNF Families, or Classes, to mimic production-like VNF connectivity, for the purposes of interopability checks.
-- Candidate VNF Validation (Termed Validation & Performance by CVC) will ensure complete interoperablity of VNF behaviour on the NFVI leverage VVP/VNFSDK test suites.  Testing ensures VNF can be spun up, modified, or removed, on the target NFVI (aka Interoperability).
+Exhaustive validation of CNTT reference implementations that ensure predicatble behavior and high quality of CNTT references are out of scope of RC and is the responsibility of RI. Validation of the RI may include smoke tests, soak tests, security tests and many other aspects of proficiency relevant to Cloud Infrastructure including criteria and tests dveloped in upstream projects. 
 
-**Different Distributions**
-The three step methodology described above of verifying Manifest compliance, executing Empirical Golden VNF transactions, and performing Interopability Testing is the same validation process regardless of the Distribution used to establish a cloud topology, and the components and services used in the client software stack.  
+CNTT specifications focus on Cloud Infrastructure and therefore RC is predominantly concerned with the behaviour of infrastructure elements and how infrastructure resources are consumed by VNF/CNFs. 
 
 <a name="1.1.1"></a>
-### 1.1.1 Terminology
+**Terminology**
 
 Terminology in this document will follow [CNTT Terminology](../../../ref_model/chapters/glossary.md).
 
-<a name="1.2"></a>
+<a name="1.1.2"></a>
 ## 1.2 Scope
 
 <p align="center"><img src="../figures/rc1_scope.png" alt="scope" title="RC1 Scope" width="100%"/></p>
 <p align="center"><b>Figure 1-1:</b> RC1 Scope.</p>
 
-This document covers the realisation aspects of conformance of both NFVI and VNFs. The document will cover the following topics:
+This document covers conformance of Cloud Infrastructure and VNFs with respec to their consumption of infrastructure resources. The document will cover the following topics:
 
-- Identify in details the Requirements for conformance Framework.
-- Identify in details the Requirement of Test Cases (and mapping them to requirements from The Reference Model and The OpenStack Based Reference Architecture ).
-- Analysis of existing community projects.
-- Propose an E2E Framework for conformance of NFVI and VNFs.
-- Playbook of instructions, user manuals, steps of how to perform verification and conformance for both NFVI and VNFs using the proposed E2E Framework.
-- Gap analysis to identify where the Gaps are in the industry (tooling, test cases, process, etc).
-- Identify development efforts needed to address any gaps identified.
+- Identify requirements for conformance framework.
+- Identify requirements of conformance tests mapped to the Reference Model and The OpenStack Based Reference Architecture.
```

Comment:
@fuqiao123 In RA-1 Chapter 02, each requirement  will point to RC sections relevant to that requirement; it will not list the actual tests. It is likely that since RC may list a set of tests within each section, a number of RA- requirements will point to the same section. A more granular mapping will have to be part of RC. But we can discuss if you have other ideas.